core/next.config.js

```diff
@@ -22,6 +22,18 @@ const nextConfig = {
     ignoreDuringBuilds: !!process.env.CI,
     dirs: ['app', 'client', 'components', 'lib', 'middlewares'],
   },
+  webpack: (config) => {
```

Comment:
This change is ineffective and won't impact anything unless we pass additional flags to update the devtool packaging during the build.